### ISSUE TITLE
linking fails on windows

### ISSUE THREAD

**Issue by ThePhilderbeast** [biozic/d2sqlite3#47]
when linking an app to d2sqlite3 on windows using the "all included" config the linker will fail to find sqlite3.lib causing the build to fail

```
Linking...
OPTLINK (R) for Win32  Release 8.00.17
Copyright (C) Digital Mars 1989-2013  All rights reserved.
http://www.digitalmars.com/ctg/optlink.html
sqlite3.lib
 Warning 2: File Not Found sqlite3.lib
```

**Comment by biozic** [comment]
Yes, the "all included" option is rather experimental, since I don't use Windows. See https://stackoverflow.com/questions/36332219 for a possible fix.

**Comment by rjframe** [comment]
The problem here is https://github.com/dlang/dub/issues/1458

If you copy the dll and lib files manually to your own project it will work.

**Comment by ThePhilderbeast** [comment]
I dont think thats the issue here, as when i use DDBC (https://github.com/buggins/ddbc/) that dose  a similar thing it works properly

i think it might be because they have "sourceFiles-windows-x86_64" : [ "libs/win64/sqlite3.lib" ] I will have to test and confirm though

**Comment by rjframe** [comment]
DDBC is copying the files themselves.

The import libs are missing here though; PR https://github.com/biozic/d2sqlite3/pull/48

**Comment by biozic** [comment]
Fix #48
